etl/steps/data/grapher/artificial_intelligence/2024-06-28/ai_corporate_investment.py

```diff
@@ -0,0 +1,47 @@
+"""Load a garden dataset and create a grapher dataset."""
```

Comment:
I would personally prefer to run several of these steps in garden, and leave the essential for grapher, but it's OK if it works for you

Reply:
felt like this made sense here given that I am creating these grapher steps separately specifically bc of how I want to plot the data!